ansible/idr-mineotaur.yml

```diff
@@ -0,0 +1,27 @@
+# Setup IDR Mineotaur server
+- hosts: "{{ idr_environment | default('idr') }}-a-dockermanager-hosts"
+
+  tasks:
+
+  - name: download mineotaur data
+    become: yes
+    synchronize:
+      src: "rsync://{{ TBD_omero_host }}/mineotaur/"
+      dest: "/tmp/mineotaur/"
+
+  - name: change mineotaur to 1000 uid
+    become: yes
+    file:
+      path: "/tmp/mineotaur"
+      owner: 1000
+
+  - name: enable mineotaur container
+    docker_container:
+      name: mineotaur
+      image: imagedata/mineotaur
```

Comment:
You've added a `TBD_omero_host` above, so how about a `# TODO: TBD_version` as a reminder to tag the docker image in future?